### PR TITLE
Add namespace in helm chart

### PR DESCRIPTION
**Pull request by hanzala1234**

### CODE REVIEW

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/fbce9cceefb6e3f476d0cf92e8be7ba345268876/checks) has Failed!

**Review by abdulhaseeb2 — DISMISSED** [review]
lgtm

File: application/values-test.yaml [hunk 1]
```diff
@@ -322,12 +322,21 @@ space:
   additionalLabels:
     key: value
 
+
```
Comment (rasheedamir) [inline application/values-test.yaml:325]: why empty extra line?

File: application/values-test.yaml [hunk 2]
```diff
@@ -322,12 +322,21 @@ space:
...
   tenant: "dominator-test"
 
+namespace:
+  enabled: true
+  # Additional labels on space
+  additionalLabels:
+    key: value
+    stakater.com/tenant: tenant-alpha
+
+
```
Comment (rasheedamir) [inline application/values-test.yaml:339]: why two empty extra lines?

File: README.md [hunk 3]
```diff
@@ -124,6 +124,9 @@ To uninstall the chart:
 | space.additionalLabels | Additional labels for Space Custom Resource | `{}` |
 | space.annotations | Annotations for Space Custom Resource | `{}` |
 | space.tenant | Tenant associated with Space Custom Resource | `""` |
+| namespace.enabled | Enable Space Custom Resource | `false` |
```
Comment (rasheedamir) [inline README.md:127]: wrong description

File: README.md [hunk 4]
```diff
@@ -124,6 +124,9 @@ To uninstall the chart:
...
+| namespace.additionalLabels | Additional labels for Space Custom Resource | `{}` |
+| namespace.annotations | Annotations for Space Custom Resource | `{}` |
```
Comment (rasheedamir) [inline README.md:129]: all wrong descriptions! it's namespace not space

**Review by rasheedamir — CHANGES_REQUESTED** [review]
fix the comments

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/672eb3eebb985f46d0bc33c9bd8e3c7a0a8bc527/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/cca1fc93e3376666d18a08803746b54edf07a768/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/925743cd19f8204a26ba25d97a223e9884a11f3d/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Validation Successful

**Review by bilalbokharee — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (hanzala1234) [reply README.md]: fixed

**Review by abdulhaseeb2 — APPROVED** [review]
LGTM

**Review by rasheedamir — APPROVED** [review]

**Pull request merged**